### PR TITLE
MongoDB project - Astrid

### PR DESCRIPTION
**Pull request by Aepera**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,37 +1,119 @@
...
+  try {
+    const findTitle = await NetflixTitle.findOne({ _id: id })
+    if (findTitle) {
+      res.json(findTitle)
+    } else {
+      res.status(404).json({ error: 'id not found' })
+    }
+  } catch (error) {
+    res.status(400).json({ error: 'request not valid' })
+  }
```
Comment (Andrealarsson) [inline server.js:83]: Clean and understandable errors 👍

File: server.js [hunk 2]
```diff
@@ -1,37 +1,119 @@
...
+dotenv.config()
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
+const NetflixTitle = mongoose.model('NetflixTitle', {
```
Comment (Andrealarsson) [inline server.js:14]: Nice and structured code 😃

**Review by Andrealarsson — APPROVED** [review]
Great work with this project! It is easy to read, understand and the code looks clean! 💯 🙌

File: README.md [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
+
+get('/titles/title/:title') param for title
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+get('/titles/country/:country') param for country, displays all titles from this country
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+API: https://mongo-netflix.herokuapp.com/
```
Comment (KarolCislowski) [inline README.md:18]: nice short documentation 👍🏻

File: server.js [hunk 4]
```diff
@@ -1,37 +1,119 @@
...
+  show_id: Number,
```
Comment (KarolCislowski) [inline server.js:15]: You will be provided with _id so it is no need to pass this one 🤓

File: server.js [hunk 5]
```diff
@@ -1,37 +1,119 @@
...
+  const { title } = req.query
+
+  if (title) { 
+    const titleRegex = new RegExp(title, 'i') 
+    const titles = await NetflixTitle.find({ title: titleRegex })
+    res.json(titles)
+  } else {
+    const titles = await NetflixTitle.find()
+    res.json({ length: titles.length, data: titles })
+  }
```
Comment (KarolCislowski) [inline server.js:69]: nice filtering

**Review by KarolCislowski — APPROVED** [review]
Nice API 👍🏻

**Pull request closed without merging**